repo/packages/B/bookkeeper/0/marathon.json.mustache

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "{{bookkeeper.app_id}}",
+  "cpus":{{bookkeeper.cpus}},
+  "mem": {{bookkeeper.mem}},
+  "instances": {{bookkeeper.instances}},
```

Comment:
since this is scalable .. does it make sense to enforce a "hostname:UNIQUE" constraint?